src/support/azext_support/_help.py

```diff
@@ -0,0 +1,204 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+
+helps['support'] = """
+    type: group
+    short-summary: Manage and administer support tickets.
+"""
+
+helps['support services'] = """
+    type: group
+    short-summary: Commands to manage services.
+"""
+
+helps['support services list'] = """
+    type: command
+    short-summary: List services.
+    examples:
+      - name: List services available for creating support ticket.
+        text: |-
+               az support services list
+"""
+
+helps['support services show'] = """
+    type: command
+    short-summary: Show service.
+    examples:
+      - name: Show service named 'Virtual Machine Running Windows Service'.
+        text: |-
+               az support services show --service-name "6f16735c-b0ae-b275-ad3a-03479cfa1396"
+"""
```

Comment:
What does it mean this string "6f16735c-b0ae-b275-ad3a-03479cfa1396"? Maybe a dummy name like `MyWindowsServiceID` to direct user where to find this value would be appropriate?

Reply:
Addressed in latest commit.